### ISSUE TITLE
checkout step for neume-network/core wrongfully assumes that if neume-network/data@branchname is run, that branch is available on neume-network/core@branchname too

### ISSUE THREAD

**Issue by TimDaub** [neume-network/data#28]
- https://github.com/neume-network/data/blob/c05e8eafc8f2b078096b8a7d088fab7d725c3b64/.github/actions/run_step/action.yml#L39

**Comment by il3ven** [comment]
I had done this change because I was testing everything on dev branch. I was using dev brach on data, core, strategies and all other packages. To test I had to change the branch name in `action.yml` and then before merging the PR change it again. To prevent this step I made the change to use the same branch name.

**Comment by TimDaub** [comment]
makes sense, and it worked well. I'm actually not even sure what would be a better solution at this point...
